resources/definitions/winbo_dragonl4.def.json

```diff
@@ -0,0 +1,137 @@
+{
+    "version": 2,
+    "name": "Winbo FDM-Dragon(L)4",
```

Comment:
I forgot this in the previous review. We are not allowing to use **FDM** in the printer names. You can choose for instance **FFF** standing for Fused Filament Fabrication. We recently change our **Custom FDM Printer** to be **Custom FFF Printer**. Of course you can also choose another different name that doesn't contain the acronym.